bluetooth-audio/bluetooth-udev

```diff
@@ -16,10 +16,16 @@ add)
     echo $NAME | tr -d '"' > /var/cache/bluetooth/reconnect_device  # Save the mac address of the last connected device (for reconnect after reboot)
     mplayer -volume $CONNECTION_NOTIFY_VOLUME /usr/src/sounds/connect.mp3 -really-quiet > /dev/null 2>&1
     amixer sset PCM,0 $SYSTEM_OUTPUT_VOLUME% > /dev/null &
+    if [[ "$BLUETOOTH_SCRIPTS" == true ]]; then
```

Comment:
I would change this line to check if the variable is defined or not, so something like this:

```bash
if [[ -z "$BLUETOOTH_SCRIPTS" ]]; then
   python /usr/src/bluetooth-scripts/bluetooth-connect
fi
```

Reply:
Same with the disconnect case.